docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,200 @@
+       privateKeySecretRef:
+         name: letsencrypt-staging-secret
+         namespace: default
+       domains:
+         include:
+           - mydomain.com # The main domain. Edit this value
+           - my.sub1.mydomain.com # The subdomain provided in the DNS Entry created in the previous step. Edit this value
+           - "*.mydomain.com" # A wildcard DNS record. Edit this value
+         # Optionally, restrict domain ranges for which certificates can be requested
+   #     exclude:
+   #       - my.sub2.mydomain.com # Edit this value
+   EOF
+   ```
+
+6. Create a Certificate CR. See the following example and modify values of the **spec.secretName**, **spec.commonName**, and **spec.issuerRef.name**, and **spec.dnsNames** parameters. As the value for the **spec.issuerRef.name** parameter, use the value from te **metadata.name** parameter of the Issuer CR. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: cert.gardener.cloud/v1alpha1
+   kind: Certificate
+   metadata:
+     name: httpbin-cert
+     namespace: istio-system
+   spec:
+     secretName: httpbin-tls-credentials # Name of the Secret created using this CR. Edit this value
+     commonName: mydomain.com # The main domain. Edit this value
+     issuerRef:
+       name: letsencrypt-staging # Name of the Issuer created in the previous step. Edit this value
+       namespace: default
+     dnsNames:
+       - my.sub1.mydomain.com # Edit this value
+   EOF
+   ```
+
+7. Create a Gateway CR. See the following example and modify values of the **spec.servers.tls.credentialName** and **spec.servers.hosts** parameters. For the **spec.servers.tls.credentialName** parameter, use the **spec.secretName** value of the Certificate CR. As the value of **spec.servers.hosts**, use the subdomain from the **spec.dnsName** parameter of the DNSEntry CR. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: networking.istio.io/v1alpha3
+   kind: Gateway
+   metadata:
+     name: httpbin-gateway
+   spec:
+     selector:
+       istio: ingressgateway # Use Istio Ingress Gateway as deafult
+     servers:
+       - port:
+           number: 443
+           name: https-httpbin
+           protocol: HTTPS
+         tls:
+           mode: SIMPLE
+           credentialName: secret-name # Name of the Secret created in the Certificate CR in the previous step. Edit this value
+         hosts:
+           - "my.sub1.mydomain.com" # Edit this value
+   EOF
+   ```
+
+8. Add your subdomain(s) at the end of the API Gateway **domain.allowlist** parameter. See the following example: `--domain-allowlist=my.sub1.mydomain.com`. Use a comma as a separator. Run the following command, to edit the deployment:
+
+   ```bash
+  kubectl edit deployment -n kyma-system api-gateway
+  ```
+
+   Press `i` to enter and `esc` to exit the interactive mode. Save the changes and exit the editor by pressing `:wq`.
+
+9. Restart the relevant deployment using this command:
```

Comment:
Previous edit will impact in rollout of API Gateway deployment.
I guess it is not required step